### DISCUSSION TITLE
Add the Minecraft Launcher "News" tab

### DISCUSSION THREAD

**Discussion by ghost** [PrismLauncher/PrismLauncher#193] [Ideas]
The Official Minecraft launcher has a "News" tab you can access to see the most frequent News about Minecraft. My idea for importing this is:
- Only display Java News
- Display Date Posted
- Like Modrinth and CurseForge tabs for installing mods, make it so you can read the News article, whatever that may be, though the launcher.

I'm not even sure if this is possible, but just an idea that I thought of :D

**Comment by DioEgizio** [comment]
i already suggested this in polymc times.
it's probably possible

**Comment by DioEgizio** [comment]
also, please use issues for this type of stuff. made an issue now in #194

**Reply by ghost** [reply]
oh my bad haha.
